users/api/requests.go

```diff
@@ -16,10 +18,19 @@ type createClientReq struct {
 	token  string
 }
 
-func (req createClientReq) validate() error {
+func (req createClientReq) validate(passRegex *regexp.Regexp) error {
```

Comment:
Let's use package-level variable for regex that we set in `MakeHandler` and use it directly, rather than passing it as a parameter.